### PR TITLE
Rename QoS*Policy enum's to *Policy

### PR DESCRIPTION
**Pull request by suab321321**
Renamed QoS*Policy enum's to *Policy #371
I have made the necessary changes to the which was to rename the class name of few classes which was starting by name of QoS.

### CODE REVIEW

File: rclpy/rclpy/qos.py [hunk 1]
```diff
@@ -270,8 +270,11 @@ class QoSHistoryPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_HISTORY_KEEP_ALL = 2
     KEEP_ALL = RMW_QOS_POLICY_HISTORY_KEEP_ALL
 
+#  Naming the class HistoryPolicy QoSHistoryPolicy 
+#  because it is used in other places with the "Qos" infront
+QoSHistoryPolicy=HistoryPolicy
```
Comment (sloretz) [inline rclpy/rclpy/qos.py]: I think the whitespace on this line will fail the linter test. It should be `QoSHistoryPolicy=HistoryPolicy` instead, and same a few places below.

File: rclpy/rclpy/qos.py [hunk 2]
```diff
@@ -270,8 +270,11 @@ class QoSHistoryPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_HISTORY_KEEP_ALL = 2
     KEEP_ALL = RMW_QOS_POLICY_HISTORY_KEEP_ALL
 
+#  Naming the class HistoryPolicy QoSHistoryPolicy 
+#  because it is used in other places with the "Qos" infront
```
Comment (sloretz) [inline rclpy/rclpy/qos.py]: Might not need the comment. I think it's clear enough without it.

File: rclpy/rclpy/qos.py [hunk 3]
```diff
@@ -285,13 +288,18 @@ class QoSReliabilityPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_RELIABILITY_BEST_EFFORT = 2
     BEST_EFFORT = RMW_QOS_POLICY_RELIABILITY_BEST_EFFORT
 
+# Naming the class ReliabilityPolicy QoSReliabilityPolicy 
+# because it is used in other places with the "Qos" infront
+QoSReliabilityPolicy=ReliabilityPolicy
 
-class QoSDurabilityPolicy(QoSPolicyEnum):
+
+
```
Comment (sloretz) [inline rclpy/rclpy/qos.py]: Looks like there's 3 blank lines, the linter will probably complain that it should only be 2 blank lines between top level definitions

File: rclpy/rclpy/qos.py [hunk 4]
```diff
@@ -285,13 +288,18 @@ class QoSReliabilityPolicy(QoSPolicyEnum):
...
+class DurabilityPolicy(QoSPolicyEnum):
     """
     Enum for QoS Durability settings.
 
     This enum matches the one defined in rmw/types.h
-    """
+    """
```
Comment (sloretz) [inline rclpy/rclpy/qos.py]: Please remove the trailing whitespace

File: rclpy/rclpy/qos.py (hunk 2 above)
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]: Maybe, something like: `Alias with the old name, for retrocompatibility` is clearer.

File: rclpy/rclpy/qos.py [hunk 5]
```diff
@@ -285,13 +288,18 @@ class QoSReliabilityPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_RELIABILITY_BEST_EFFORT = 2
     BEST_EFFORT = RMW_QOS_POLICY_RELIABILITY_BEST_EFFORT
 
+# Naming the class ReliabilityPolicy QoSReliabilityPolicy 
+# because it is used in other places with the "Qos" infront
```
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]: same

File: rclpy/rclpy/qos.py [hunk 6]
```diff
@@ -316,7 +327,10 @@ class QoSLivelinessPolicy(QoSPolicyEnum):
     MANUAL_BY_NODE = RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_NODE
     RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_TOPIC = 3
     MANUAL_BY_TOPIC = RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_TOPIC
-
+    
+# Naming the class DurabilityPolicy QoSDurabilityPolicy 
+# because it is used in other places with the "Qos" infront
+QoSDurabilityPolicy=DurabilityPolicy
```
Comment (sloretz) [inline rclpy/rclpy/qos.py]: Did you mean `LivelinessPolicy` here? It looks like `DurabilityPolicy` is above, and the class name for `QoSLivelinessPolicy` should be changed as well.

File: rclpy/rclpy/qos.py (hunk 1 above)
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]: spaces around `=` (linters need them to pass).

File: rclpy/rclpy/qos.py [hunk 7]
```diff
@@ -285,13 +288,18 @@ class QoSReliabilityPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_RELIABILITY_BEST_EFFORT = 2
     BEST_EFFORT = RMW_QOS_POLICY_RELIABILITY_BEST_EFFORT
 
+# Naming the class ReliabilityPolicy QoSReliabilityPolicy 
+# because it is used in other places with the "Qos" infront
+QoSReliabilityPolicy=ReliabilityPolicy
```
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]: spaces around `=`

File: rclpy/rclpy/qos.py [hunk 8]
```diff
@@ -334,20 +348,21 @@ def __init__(self, qos_profile: QoSProfile, profile_name: str):
         self.name = profile_name
 
 
-_qos_profile_default = QoSProfile(**_rclpy.rclpy_get_rmw_qos_profile('qos_profile_default'))
+_qos_profile_default = _rclpy.rclpy_get_rmw_qos_profile(
```
Comment (sloretz) [inline rclpy/rclpy/qos.py]: Please undo this change and the changes below. I these variables being `QoSProfile` instances is desired.

File: rclpy/rclpy/qos.py (hunk 3 above)
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]: Delete the two extra new lines above.

File: rclpy/rclpy/qos.py (hunk 4 above)
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]: Delete trailing spaces

File: rclpy/rclpy/qos.py [hunk 9]
```diff
@@ -300,14 +308,17 @@ class QoSDurabilityPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_DURABILITY_VOLATILE = 2
     VOLATILE = RMW_QOS_POLICY_DURABILITY_VOLATILE
 
+# Naming the class DurabilityPolicy QoSDurabilityPolicy 
+# because it is used in other places with the "Qos" infront
```
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]: same

File: rclpy/rclpy/qos.py [hunk 10]
```diff
@@ -300,14 +308,17 @@ class QoSDurabilityPolicy(QoSPolicyEnum):
...
+QoSDurabilityPolicy=DurabilityPolicy
```
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]: spaces around `=`

File: rclpy/rclpy/qos.py [hunk 11]
```diff
@@ -300,14 +308,17 @@ class QoSDurabilityPolicy(QoSPolicyEnum):
...
 
 class QoSLivelinessPolicy(QoSPolicyEnum):
     """
     Enum for QoS Liveliness settings.
 
     This enum matches the one defined in rmw/types.h
     """
-
+
```
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]: Delete trailing spaces

File: rclpy/rclpy/qos.py [hunk 12]
```diff
@@ -300,14 +308,17 @@ class QoSDurabilityPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_DURABILITY_VOLATILE = 2
     VOLATILE = RMW_QOS_POLICY_DURABILITY_VOLATILE
 
+# Naming the class DurabilityPolicy QoSDurabilityPolicy 
+# because it is used in other places with the "Qos" infront
+QoSDurabilityPolicy=DurabilityPolicy
 
 class QoSLivelinessPolicy(QoSPolicyEnum):
```
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]: I think you forgot this one.

File: rclpy/rclpy/qos.py (hunk 6 above)
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]: update comment, and spaces around `=`.

File: rclpy/rclpy/qos.py [hunk 13]
```diff
@@ -334,20 +348,21 @@ def __init__(self, qos_profile: QoSProfile, profile_name: str):
...
+qos_profile_sensor_data = _rclpy.rclpy_get_rmw_qos_profile(
+    'qos_profile_sensor_data')
+qos_profile_services_default = _rclpy.rclpy_get_rmw_qos_profile(
+    'qos_profile_services_default')
+qos_profile_parameters = _rclpy.rclpy_get_rmw_qos_profile(
+    'qos_profile_parameters')
+qos_profile_parameter_events = _rclpy.rclpy_get_rmw_qos_profile(
+    'qos_profile_parameter_events')
+qos_profile_action_status_default = _rclpy_action.rclpy_action_get_rmw_qos_profile(
+    'rcl_action_qos_profile_status_default')
```
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]: Please undo this change and the similar changes above. `QoSProfile(**...)` was ok.

**Comment by ivanpauno** [comment]
Ups, @sloretz didn't see your review when I started

**Comment by suab321321** [comment]
I will commit the requested changes

File: rclpy/rclpy/qos.py (hunk 8 above)
Comment (suab321321) [reply rclpy/rclpy/qos.py]: I have not done these changes..Anyways I will try to reslove this
